install.sh

```diff
@@ -187,8 +187,8 @@ _check_dep() {
 	#Checks dependencies
 
 	echo "$_BOLD * Checking dependencies...$_NORMAL"
-	echo -n "   * Nautilus ............................ "
```

Comment:
This is actually the only thing I had to change to get this working on MATE. With this thumbnailer having already been ported to the new freedesktop thumbnailer spec, it is pretty much DE-agnostic now. Hence why I hesitated to re-add dconf integration.